app/javascript/components/supplemental_application/sections/LeaseInformatonInputs.js

```diff
@@ -33,10 +33,26 @@ const getTotalMonthlyRent = (values) => {
   }
 }
 
+const toggleNoPreferenceUsed = (formApi) => {
+  formApi.values.no_preference_used = !formApi.values.preference_used
+}
+
 const LeaseInformationInputs = ({ formApi, store }) => {
   const { availableUnits } = store
   const availableUnitsOptions = formUtils.toOptions(map(availableUnits, pluck('id', 'unit_number')))
   const totalMonthlyRent = getTotalMonthlyRent(formApi.values)
+  // FIXME: will this dynamically update when the preferences are updated?
+  const preferences = store.application.preferences
+  const confirmedPreferences = filter(preferences, { 'post_lottery_validation': 'Confirmed' })
+  const confirmedPreferenceOptions = formUtils.toOptions(map([{'id': null, 'preference_name': 'None'}, ...confirmedPreferences], pluck('id', 'preference_name')))
```

Comment:
I just changed the `no_preferenced_used` string to `null` so that the value doesn't need to be changed to null on submit. I would vote to change the name of 'id' to 'value', but the functionality still works either way.